.github/workflows/checks-for-testing-failures.yml

```diff
@@ -0,0 +1,34 @@
+name: "checks for testing failure (3sec & 5sec)"
```

Comment:
What's this for?

Reply:
I've been using these 4 checks (2, 3, 4, and 5 sec) to verify the behavior of this action in PRs.

I'd prefer to keep these around for now, as they've been useful thus far for testing, and I'm hoping to eventually leverage them for integration testing (similar pattern to what you showed me last week).

Let me know if you have any concerns with this!